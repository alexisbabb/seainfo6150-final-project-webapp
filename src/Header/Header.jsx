import React from "react";
import { Link } from "react-router-dom";
import styles from "./Header.module.css";

const Header = () => {
    return(
        <nav className = {styles.nav}>
          <Link to="/">
            <img
              className={styles.companyLogo} src="https://delawarehumane.org/wp-content/uploads/2016/07/DHA-Web-Logo-For-Footer.png" alt="Delaware Humane Association Logo"/>
          </Link>
          <Link className={styles.navigation} to="/adoptionprocess">Adoption Process Overview</Link>
          <Link className={styles.navigation} to="/petlist">Adoptable Pets</Link>
          <Link className={styles.navigation} to="/foodpantry">Food Pantry</Link>
          <Link className={styles.navigation} to="/clinicservices">Clinic Services</Link>
        </nav>
    )
};

export default Header;