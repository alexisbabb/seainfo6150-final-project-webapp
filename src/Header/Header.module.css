nav {
    display: flex;
    align-items: center;
    background-color: #b8610f; 
    padding-left: 0.5em;  
  }
  
  .companyLogo{
    width :60px;  
  }


  .navigation{
      text-decoration: none;
      font-family: 'Helvetica', sans-serif;
      color: white; 
      font-weight: bold; 
      font-size: 10px; 
      padding: 1.5em;  

    }
  
  @media screen and (min-width: 600px) {
    .companyLogo {
      width: 80px;
    }

    .navigation {
      font-size: 12px;
      padding: 1.5em; 
    }
  }
  
  @media screen and (min-width: 800px) {
    .companyLogo {
      width: 120px;
    }

    .navigation {
      font-size: 16px;
      padding: 1.5em; 
    }
  }
  
  @media screen and (min-width: 1200px) {
    .companyLogo {
      width: 160px;
    }
    .navigation {
      font-size: 20px;
      padding: 1.5em; 
    }
  }
  
