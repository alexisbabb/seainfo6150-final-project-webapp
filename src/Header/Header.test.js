import React from "react";
import Header from "./Header"

describe("Header tests", () => {
    it("renders correctly", () => {
        const src = "https://delawarehumane.org/wp-content/uploads/2016/07/DHA-Web-Logo-For-Footer.png";
        const { container } = render(<Header src={src}/>);
        expect(container).toMatchSnapshot();
    });
});