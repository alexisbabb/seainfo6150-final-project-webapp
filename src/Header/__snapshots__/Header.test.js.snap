// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests renders correctly 1`] = `
<div>
  <nav
    class="nav"
  >
    <a
      href="/"
    >
      <img
        alt="Delaware Humane Association Logo"
        class="companyLogo"
        src="https://delawarehumane.org/wp-content/uploads/2016/07/DHA-Web-Logo-For-Footer.png"
      />
    </a>
    <a
      class="navigation"
      href="/adoptionprocess"
    >
      Adoption Process Overview
    </a>
    <a
      class="navigation"
      href="/petlist"
    >
      Adoptable Pets
    </a>
    <a
      class="navigation"
      href="/foodpantry"
    >
      Food Pantry
    </a>
    <a
      class="navigation"
      href="/clinicservices"
    >
      Clinic Services
    </a>
  </nav>
</div>
`;
